controllers/sandbox_controller.go

```diff
@@ -209,7 +209,6 @@ func (r *SandboxReconciler) computeReadyCondition(sandbox *sandboxv1alpha1.Sandb
 	if podReady && svcReady {
 		readyCondition.Status = metav1.ConditionTrue
 		readyCondition.Reason = "DependenciesReady"
-		return readyCondition
```

Comment:
While removing the early return is a valid stylistic choice for creating a single exit point, the condition for `Ready=False` is now less explicit. Consider adding an `else` block to set a `Reason` such as `"DependenciesNotReady"`. This would make the sandbox's status clearer during inspection.